pom.xml

```diff
@@ -9,13 +9,13 @@
 	<parent>
 		<groupId>org.opentestsystem.shared</groupId>
 		<artifactId>shared-build-parent</artifactId>
-		<version>4.0.0.RELEASE</version>
+		<version>4.0.3</version>
 	</parent>
     
     <properties>
         <bouncycastle.version>1.50</bouncycastle.version>
-	<progman-client.version>4.0.0.RELEASE</progman-client.version>
-    </properties>
+		<progman-client.version>4.0.1-SNAPSHOT</progman-client.version>
```

Comment:
Please update this to use the latest released version of progman-client which also has your changes in the PR